src/main/scala/rps/Main.scala

```diff
@@ -15,65 +15,45 @@ import wiro.Config
 import wiro.server.akkaHttp._
 import wiro.server.akkaHttp.FailSupport._
 
-@enum trait Move {
-  object Rock
-  object Paper
-  object Scissors
-}
-
-@enum trait Result {
-  object Win
-  object Lose
-  object Draw
-}
-
 import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
 import wiro.Config
 import wiro.server.akkaHttp._
 
+import rps.GameApi
```

Comment:
use the same package instead of importing
`package rps` at the beginning of file